libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -203,18 +215,30 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
         this._useForm = coerceBooleanProperty(value);
     }
 
+    /**
+     * specify the column layout in the format `XLn-Ln-Mn-Sn` where n is the number of columns and can be different for each size.
+     * eg: XL2-L2-M2-S1 would create 2-column layouts for XL, L, and M sizes and single-column layout for S size.
+     */
+    @Input()
+    columnLayoutType: string;
+
```

Comment:
Let' maybe rename this  `columnLayoutType` as it sounds strange its not really a type, we are not going to set different types. What about just `column Layout`